### ISSUE TITLE
ТЗ и ИЧ

### ISSUE THREAD

**Issue by RexuMiller** [RexuMiller/auto-notification-system#1]
Сделать техническое задание и исследовательскую часть

**Comment by RexuMiller** [comment]
[ИСиТЗ.docx](https://github.com/RexuMiller/-/files/2394295/default.docx)
Вот что готово на сегодняшний день

**Comment by wldmr-key** [comment]
ok замечания:

текст написан очень небрежно и явно не прочитан. Текст имеет мало общего с тем, что обсуждалось. Где сценарии? Нужно очень сильно дорабатывать

**Comment by RexuMiller** [comment]
[ИСиТЗ.docx](https://github.com/RexuMiller/auto-notification-system/files/2417371/default.docx)

**Comment by wldmr-key** [comment]
Уже лучше. Но по-прежнему проблемы с русским языком и с четкостью изложения существенных моментов

Цели: вряд ли кредитная история ведется в твоей системе. Можешь сформулировать более понятные для банкиров цели: побольше заработать+поменьше потерять. За счет чего?

Краткие сведения об объекте автоматизации - уж слишком краткие. Мотивировка никакая. Как организован процесс голосового оповещения без твоей системы? Какие при этом есть проблемы?

Функции: мало и непонятно. Что такое "журнал изменений"? Что это за "возможность"? -  система мешать звонить что-ли не будет? Синтезатор служит для синтеза...

Состав и содержание работ по созданию (развитию) системы - не соответствуют ГОСТ - прочитай, что там требуется

> •	Обучение учащихся работе с проектируемой системой

это за гранью добра и зла... ))

----

исследовательская часть - та же проблема - нет четкого описания предметной области - Ты сразу переходишь к описанию системы. Описание скриптов - наполовину правильное, но неполное - не отражена их вариативность.

**Comment by RexuMiller** [comment]
[ИСиТЗ.docx](https://github.com/RexuMiller/auto-notification-system/files/2439419/default.docx)

**Comment by wldmr-key** [comment]
описание предметной области показывает, что ты понимаешь существо задачи. Почему же этого нет в ТЗ?!
Цель - обеспечить своевременность выплат по кредитам и информирование клиентов, сократив затраты на операторов колл-центра
Объект автоматизации - это банк, а не твоя программа. Из этого следует исходить в "кратком описании"
Вот всего-то эти две вещи и нужно исправить в ТЗ

**Comment by wldmr-key** [comment]
исследование - описание хорошее, но не завершенное. Не описан процесс обработки результатов обзвона
концептуальная схема - это не концептуальная схема
нужно ускоряться...

**Comment by RexuMiller** [comment]
[ИСиТЗ.docx](https://github.com/RexuMiller/auto-notification-system/files/2462551/default.docx)

**Comment by wldmr-key** [comment]
ты по-прежнему собираешься меня мучить просмотрами мегатекста?! Пойми, чем больше букв я вижу, тем хуже я их понимаю. Если тебя не очень интересует мое мнение о твоих изменениях, то такой подход понятен. В противном случае, старайся доставлять информацию в более удобной для меня форме

По-прежнему
>концептуальная схема - это не концептуальная схема
нужно ускоряться...

**Comment by RexuMiller** [comment]
переделал концептуальную схему
[концептуальная схема.docx](https://github.com/RexuMiller/auto-notification-system/files/2485072/default.docx)
Если это не то, то можете пояснить что я делаю не так?

**Comment by wldmr-key** [comment]
нет, совсем не так. В среду на занятии я расскажу о том, как рисуются er-диаграммы. Вы не одиноки в своих заблуждениях ))
